### PR TITLE
Add URL to Uniswap v3 contracts to smart contracts

### PR DESCRIPTION
**Pull request by horacio**
## Description

This small change just points the docs to the latest Uniswap contracts.

### CODE REVIEW

**Comment by corwintines** [comment]
Curious what others think. I know that Uniswap v2 is still the primarily used version, but always down for pointing to latest release too.

**Comment by minimalsm** [comment]
@corwintines @horacio hmmm I'd be in favour of using v3 but we already have a [Uniswap-v2 contract walkthrough tutorial](https://ethereum.org/en/developers/tutorials/uniswap-v2-annotated-code/) on the site. Why don't we keep v2 and leverage this tutorial here?

I think @qbzzt said that V3 contracts were markedly more complex/difficult to understand. Any thoughts on this discussion Ori?

**Comment by qbzzt** [comment]
> @corwintines @horacio hmmm I'd be in favour of using v3 but we already have a [Uniswap-v2 contract walkthrough tutorial](https://ethereum.org/en/developers/tutorials/uniswap-v2-annotated-code/) on the site. Why don't we keep v2 and leverage this tutorial here?
> 
> I think @qbzzt said that V3 contracts were markedly more complex/difficult to understand. Any thoughts on this discussion Ori?

They are more difficult to document (for one thing, they are written less to be legible and more to be efficient, IIRC). However, that only applies to the annotated code in the tutorials. I see no reason to avoid linking to them when teaching composability.

File: src/content/developers/docs/smart-contracts/composability/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ incomplete: true
 
 ## A brief introduction {#a-brief-introduction}
 
-Smart contracts are public on Ethereum and can be thought of as open APIs. You don't need to write your own smart contract to become a dapp developer, you just need to know how to interact with them. For example, you can use the existing smart contracts of [Uniswap](https://uniswap.exchange/swap), a decentralized exchange, to handle all the token swap logic in your app – you don't need to start from scratch. [Check out some of their contracts](https://github.com/Uniswap/uniswap-v2-core/tree/master/contracts).
+Smart contracts are public on Ethereum and can be thought of as open APIs. You don't need to write your own smart contract to become a dapp developer, you just need to know how to interact with them. For example, you can use the existing smart contracts of [Uniswap](https://uniswap.exchange/swap), a decentralized exchange, to handle all the token swap logic in your app – you don't need to start from scratch. [Check out some of their contracts](https://github.com/Uniswap/uniswap-v3-core/tree/main/contracts).
```
Comment (corwintines) [inline src/content/developers/docs/smart-contracts/composability/index.md:11]:
```suggestion
Smart contracts are public on Ethereum and can be thought of as open APIs. You don't need to write your own smart contract to become a dapp developer, you just need to know how to interact with them. For example, you can use the existing smart contracts of [Uniswap](https://uniswap.exchange/swap), a decentralized exchange, to handle all the token swap logic in your app – you don't need to start from scratch. Check out some of their [v2](https://github.com/Uniswap/uniswap-v2-core/tree/master/contracts) and [v3](https://github.com/Uniswap/uniswap-v3-core/tree/main/contracts) contracts.
```
We could do the best of both worlds here?

**Comment by minimalsm** [comment]
@horacio bringing this in with the suggested changes from @corwintines

Thanks again for this!

**Comment by minimalsm** [comment]
@all-contributors please add @horacio for docs

**Comment by horacio** [comment]
> @horacio bringing this in with the suggested changes from @corwintines
> 
> Thanks again for this!

Thank you! ⚡

**Pull request merged**